exec/java-exec/src/main/java/org/apache/drill/exec/store/httpd/HttpdParser.java

```diff
@@ -45,26 +45,273 @@
+    LOGFIELDS.put("connection.server.name.last", "STRING:connection.server.name.last");
+    LOGFIELDS.put("connection.server.name.original", "STRING:connection.server.name.original");
+    LOGFIELDS.put("connection.server.name.last", "STRING:connection.server.name.last");
+    LOGFIELDS.put("response.connection.status", "HTTP.CONNECTSTATUS:response.connection.status");
+    LOGFIELDS.put("response.connection.status.last", "HTTP.CONNECTSTATUS:response.connection.status.last");
+    LOGFIELDS.put("response.connection.status.original", "HTTP.CONNECTSTATUS:response.connection.status.original");
+    LOGFIELDS.put("response.connection.status.last", "HTTP.CONNECTSTATUS:response.connection.status.last");
+    LOGFIELDS.put("request.bytes", "BYTES:request.bytes");
+    LOGFIELDS.put("request.bytes.last", "BYTES:request.bytes.last");
+    LOGFIELDS.put("request.bytes.original", "BYTES:request.bytes.original");
+    LOGFIELDS.put("request.bytes.last", "BYTES:request.bytes.last");
+    LOGFIELDS.put("response.bytes", "BYTES:response.bytes");
+    LOGFIELDS.put("response.bytes.last", "BYTES:response.bytes.last");
+    LOGFIELDS.put("response.bytes.original", "BYTES:response.bytes.original");
+    LOGFIELDS.put("response.bytes.last", "BYTES:response.bytes.last");
+    LOGFIELDS.put("total.bytes", "BYTES:total.bytes");
+    LOGFIELDS.put("total.bytes.last", "BYTES:total.bytes.last");
+    LOGFIELDS.put("total.bytes.original", "BYTES:total.bytes.original");
+    LOGFIELDS.put("total.bytes.last", "BYTES:total.bytes.last");
+    LOGFIELDS.put("request.cookies", "HTTP.COOKIES:request.cookies");
+    LOGFIELDS.put("request.cookies.last", "HTTP.COOKIES:request.cookies.last");
+    LOGFIELDS.put("request.cookies.original", "HTTP.COOKIES:request.cookies.original");
+    LOGFIELDS.put("request.cookies.last", "HTTP.COOKIES:request.cookies.last");
+    LOGFIELDS.put("response.cookies", "HTTP.SETCOOKIES:response.cookies");
+    LOGFIELDS.put("response.cookies.last", "HTTP.SETCOOKIES:response.cookies.last");
+    LOGFIELDS.put("response.cookies.original", "HTTP.SETCOOKIES:response.cookies.original");
+    LOGFIELDS.put("response.cookies.last", "HTTP.SETCOOKIES:response.cookies.last");
+    LOGFIELDS.put("request.user-agent", "HTTP.USERAGENT:request.user-agent");
+    LOGFIELDS.put("request.user-agent.last", "HTTP.USERAGENT:request.user-agent.last");
+    LOGFIELDS.put("request.user-agent.original", "HTTP.USERAGENT:request.user-agent.original");
+    LOGFIELDS.put("request.user-agent.last", "HTTP.USERAGENT:request.user-agent.last");
+    LOGFIELDS.put("request.referer", "HTTP.URI:request.referer");
+    LOGFIELDS.put("request.referer.last", "HTTP.URI:request.referer.last");
+    LOGFIELDS.put("request.referer.original", "HTTP.URI:request.referer.original");
+    LOGFIELDS.put("request.referer.last", "HTTP.URI:request.referer.last");
+      /*LOGFIELDS.put("request_receive_time_weekyear__utc", "TIME_YEAR:request_receive_time_weekyear__utc");
```

Comment:
Please remove this commented block (or if it needs to be uncommented in future - please leave a comment when and why this needs to be done).

Reply:
Done